Bottom: bc200acfdf479b5b5be9644d2add4bad3f5cb854
Top:    b4a9b6b400bfe4e74f8f66c8c2425da886c9b315
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 12:39:29 -0500

Refresh of ticks

---

diff --git a/farm-checkout b/farm-checkout
index 7a25464..ed1bf51 100755
--- a/farm-checkout
+++ b/farm-checkout
@@ -6,7 +6,7 @@ set -e
 cd $linuxgit
 mkdir -p $farmsrc
 for tag in $tags; do
-	srctree=$farmsrc/`echo $tag | sed 's,/,-,'`
+	srctree=$farmsrc/$(echo $tag | sed 's,/,-,')
 	echo -n "Checking out $srctree "
 	if test -d $srctree; then
 		echo -e "\e[33mEXISTS\e[0m"
diff --git a/farm-compile b/farm-compile
index b8220c4..94655fa 100755
--- a/farm-compile
+++ b/farm-compile
@@ -25,7 +25,7 @@ for bintree in $(ls -d $farmbin/* | sort -V); do
 		ccflags=-m64
 	fi
 	test -n "$arch" || continue
-	log=Log-`basename $bintree`
+	log=Log-$(basename $bintree)
 	rm -f $log $log-ok $log-error
 	$make clean KERNELRELEASE=0 VERSION_HEADER=. >/dev/null
 	echo -n "Checking $bintree "
